src/controls/geocoder.js

```diff
@@ -23,7 +23,7 @@ export default class Geocoder {
 
     // Template
     var el = document.createElement('div');
-    el.className = 'mapboxgl-ctrl-geocoder';
+    el.className = 'mapboxgl-ctrl-geocoder mapboxgl-ctrl-geocoder-directions';
```

Comment:
Is there a reason we can't use replace `mapboxgl-ctrl-geocoder` here with `mapboxgl-ctrl-geocoder-directions`?

Reply:
Actually, that was my first thought. But this will break styles for the guys who already using current class name, this why I added additional class

Reply:
But does this mean if I'm changing styles for the MapboxGeocoder I'll inadvertently also change the style used by MapboxDirections?

Reply:
Theoretically, it is possible to break style by adding something in MapboxGeocoder, but I don't think this may happen. For example `outline: 2px solid red` in MapboxGeocder styles will add a red outline for MapboxDirections because `outline` isn't overridden in MapboxDirections styles.